src/compiler/code_generator.ts

```diff
@@ -632,7 +632,16 @@ export class CodeGenerator {
 
       let idx: number;
       if (specialInitTargetAttr) {
-        idx = block!.insertData(`${fullExpression} === '${attrs[targetAttr]}'`, "attr");
+        let targetExpr = targetAttr in attrs && `'${attrs[targetAttr]}'`;
+        if (!targetExpr && ast.attrs) {
+          // look at the dynamic attribute counterpart
+          const dynamicTgExpr = ast.attrs[`t-att-${targetAttr}`];
```

Comment:
I'm not sure I understand.
I re-pushed to avoid having the t-att-value expression placed in a variable, instead, that expression is directly used to produced the checked boolean attribute